### PR TITLE
Remove yarn lock PART 3

### PR DESCRIPTION
**Pull request by smarletta**
package lock should be added later

### CODE REVIEW

**Comment by erwindon** [comment]
can you please REBASE this change, removing the parts that are already done or need to be skpped?

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 .gitignore
 yarn-error.log
 node_modules/
+.idea
```
Comment (erwindon) [inline .gitignore]: please remove this file from this commit; the `.idea` directory is already in the file `.gitignore`

File: package.json [hunk 2]
```diff
@@ -23,12 +23,26 @@
       "no-undef": 0,
       "no-unused-vars": 0,
       "no-console": 0,
-      "indent": ["error", 2],
+      "indent": [
+        "error",
+        2
+      ],
```
Comment (erwindon) [inline package.json]: please remove this this white-space-only change

File: package.json [hunk 3]
```diff
@@ -23,12 +23,26 @@
...
       "no-irregular-whitespace": 2,
       "no-whitespace-before-property": 2,
-      "no-multiple-empty-lines": ["error", { "max": 2, "maxEOF": 1 }],
+      "no-multiple-empty-lines": [
+        "error",
+        {
+          "max": 2,
+          "maxEOF": 1
+        }
+      ],
```
Comment (erwindon) [inline package.json]: please remove this this white-space-only change

File: package.json [hunk 4]
```diff
@@ -23,12 +23,26 @@
...
       "no-useless-escape": 0,
-      "no-constant-condition": ["error", { "checkLoops": false }],
+      "no-constant-condition": [
+        "error",
+        {
+          "checkLoops": false
+        }
+      ],
```
Comment (erwindon) [inline package.json]: please remove this this white-space-only change

**Comment by erwindon** [comment]
note that I have made changes. one was a fix because we were already referencing `test:unit` which is only created in the next PR (#192). the other one was a cleanup of `utils.js` based on the preview that I had from the coverage test.
When will you have this PR ready again?

**Comment by erwindon** [comment]
@smarletta @dawidmalina 
Gents, can you help me to fix the `navigation error`s in the travis output? my knowledge there is a bit limited...

**Comment by erwindon** [comment]
@smarletta I assume you wrote "annoying" in your commit comment? the idea was to simplify the PR so that all code that is already present in the base code is no longer present in the PR itself. This is typically done using rebase. The duplicate code will then disappear by itself. I did not mean to request removal of the line in `.gitignore`; just that its addition is no longer part of this PR to prevent duplication on merge.
Can you please rebase the PR so that it contains only the code related to the PR itself?

**Comment by erwindon** [comment]
@smarletta You are a bit unresponsive, are you ok? to perform the merge, I need an error-free branch to make sure the merged result is also ok. Traditionally the requester arranges that by fixing/rebasing/etc. Since I see the benefits (plural!) of your PR, I can do this myself, but that may result in many changes committed under my name.
Can you please indicate whether you will rebase your PRs, starting with this one?

**Comment by smarletta** [comment]
Hi,

Sorry. Switched my private machine to MacOS and had few issues.

Last week I tried the debase but failed... Not sure what I did wrong.

Kindly
Sebastiano

Erwin Dondorp <notifications@github.com> schrieb am Mo., 18. März 2019,
01:26:

**Comment by erwindon** [comment]
@smarletta yo do not provide a way forward... I'll take the lead for these PRs now.

**Comment by erwindon** [comment]
This PR has no more new code, all pieces have been cherry-picked.

**Pull request closed without merging**